index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Spartan:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-start">
+            <div class="main-logo">
+                <img src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/global/header/PS_logo.png"
+                    class="main-logo-icon" alt="main-logo">
+            </div>
+            <div class="navbar-items">
```

Comment:
I names I provided you might not be the best. Maybe you could use better names for the classes such as this one. (e.g. "navbar-start-items" to denote that this is the starting container of the navbar)